### ISSUE TITLE
IPv6 address not working for couchbase Library 

### ISSUE THREAD

**Issue by smithazac** [peterservice-rnd/robotframework-couchbaselibrary#2]
Hi,
I am using robotframework-couchbaselibrary 1.0.4 for using couch base API.

When I use the keyword Connect To Couchbase Bucket for ipv4 address, it works fine. The same function when i try to use for IPv6 address it give me the below error:
('Could not connect to Couchbase bucket. Error:', '<RC=0x15[DNS/Hostname lookup failed], There was a problem while trying to send/receive your request over the network. This may be a result of a bad network or a misconfigured client or server, C Source=(src/bucket.c,793)>')

I have tried passing the ipv6 address as below:

CouchDB_IP= "[fcff:3:10:219::94]"

The ipv6 address is accessible and using the ipv6 address I am able to login to the GUI with default port 8091.

CouchDB_Port= 8091

Can some one help me out on this?

**Comment by AleksandrKE** [comment]
Hello.
I created a [pull-request](https://github.com/peterservice-rnd/robotframework-couchbaselibrary/pull/3/files) with the necessary changes.
Could you check if these changes work?
To connect via ipv6, simply add the parameter "ipv6" with the value "allow".
For example: 
```
Connect To Couchbase Bucket | host_name | 8091 | bucket_name | password | alias=bucket | ipv6=allow
```

**Comment by smithazac** [comment]
Hello,
  I was out of town and came back today. Will update you on this in next
few days.

To verify this,I need to copy this file src/CouchbaseLibrary.py
<https://github.com/peterservice-rnd/robotframework-couchbaselibrary/pull/3/files#diff-d7ed18f5d4c80ed6b0d4df5dd128de77>
and try the same right?


Thanks,
Smitha.

**Comment by AleksandrKE** [comment]
I put these changes in PyPi.
You can install the updated package using:
`pip install robotframework-couchbaselibrary==1.0.5.dev1`

**Comment by smithazac** [comment]
Hi,
  I have upgraded couch base library as below:

**Comment by smithazac** [comment]
Hi,
  I have upgraded couch base library as below:

robotframework-couchbaselibrary    1.0.5.dev1 .

I still get the below error when i try to give ipv6 address. I tried giving
as below:

Connect To Couchbase Bucket    fcff:3:10:219::94    8091
<sessionbucket-name>    <password>    alias=bucket1
and
Connect To Couchbase Bucket   [fcff:3:10:219::94]    8091
<sessionbucket-name>    <password>    alias=bucket1

[root@1-9 couchDB]# pybot  test.robot
==============================================================================
Test

==============================================================================
View Document In Bucket                                               |
FAIL |
Setup failed:
('Could not connect to Couchbase bucket. Error:', '<RC=0x15[DNS/Hostname
lookup failed], There was a problem while trying to send/receive your
request over the network. This may be a result of a bad network or a
misconfigured client or server, C Source=(src/bucket.c,793)>')
------------------------------------------------------------------------------
Test                                                                  |
FAIL |
1 critical test, 0 passed, 1 failed
1 test total, 0 passed, 1 failed

Thanks,
Smitha.

**Comment by smithazac** [comment]
Hi, 
 It worked with the below option:
Connect To Couchbase Bucket    [fcff:3:10:219::94]    8091    <bucketname>    <password>    alias=bucket1  ipv6=allow
Thanks,
Smitha.

**Comment by aleskarovadi** [comment]
Fixed in commit [ba4277e](https://github.com/peterservice-rnd/robotframework-couchbaselibrary/commit/ba4277ef59c62d570cf26cfcd7cab78a1daa6a0d).
